### PR TITLE
Fixes #19980: spurious "connection_read(9): no connection!" in /var/log/rudder/ldap/slapd.log

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/19980

### CODE REVIEW

File: webapp/sources/scala-ldap/src/main/scala/com/normation/ldap/sdk/LDAPIOResult.scala [hunk 1]
```diff
@@ -34,7 +34,8 @@ sealed trait LDAPRudderError extends RudderError {
 object LDAPRudderError {
   // errors due to some LDAPException
   final case class BackendException(msg: String, cause: Throwable)  extends LDAPRudderError {
-    override def fullMsg: String = super.fullMsg + s"; cause was: ${RudderError.formatException(cause)}"
+    override def fullMsg: String = super.fullMsg + s"; cause was: ${RudderError.formatException(cause)} ;" +
+      s"you probably need to increase value ldap.maxPoolSize property in configuration file, please check the documentation"
```
Comment (VinceMacBuche) [inline webapp/sources/scala-ldap/src/main/scala/com/normation/ldap/sdk/LDAPIOResult.scala:38]: maybe we can put a link to the documentation ? maybe the embedded one ?

Comment (ncharles) [reply webapp/sources/scala-ldap/src/main/scala/com/normation/ldap/sdk/LDAPIOResult.scala:38]: i'm unsure we will effectively maintain this link in an error message :/

Comment (fanf) [reply webapp/sources/scala-ldap/src/main/scala/com/normation/ldap/sdk/LDAPIOResult.scala:38]: I think it's better with the link. It's better for the user, and even if it becomes incorrect at some point, it helps the user going at the right place. 
Not sure if we have the base url for the local doc though.

**Comment by ncharles** [comment]
PR updated with a new commit

**Review by fanf — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**